extensions/blocks/business-hours/business-hours.php

```diff
@@ -119,8 +134,8 @@ function jetpack_business_hours_render( $attributes ) {
 			}
 			$days_hours .= sprintf(
 				'%1$s - %2$s',
-				date( $time_format, $opening ),
-				date( $time_format, $closing )
+				gmdate( $time_format, $opening ),
+				gmdate( $time_format, $closing )
```

Comment:
This was necessary as it was triggering a PHPCS error that stopped me from committing.